.github/ISSUE_TEMPLATE.md

```diff
@@ -8,11 +8,11 @@ Prefer live chat? Message our engineers on our Gitter channel at https://gitter.
 
 **FEATURE REQUEST**
 
-1. Please describe the feature you are requesting.
+1. Has another user made the same request? Try searching for issues tagged with "[community-request](https://github.com/cockroachdb/cockroach/issues?q=is%3Aopen+is%3Aissue+label%3Acommunity-request)." If yes, please add :+1:  to the issue. If not, move on to step 2.
```

Comment:
Are we asking users to leave a comment with the body ":+1:" or to add a reaction to the existing issue? We should be crystal clear here, I think.

Also, perhaps we could just direct users to search rather than mentioning that label by name? What if an issue exists but was filed by a core contributor? I don't think the extra specificity in this case plays to our or our community's advantage.